google-cloud-storage/pom.xml

```diff
@@ -19,6 +19,14 @@
     <pubsub-proto.version>1.123.5</pubsub-proto.version>
   </properties>
   <dependencies>
+    <dependency>
+      <groupId>com.fasterxml.jackson.dataformat</groupId>
+      <artifactId>jackson-dataformat-xml</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+    </dependency>
```

Comment:
The dependency build is failing because this is declared, but not used. Can we remove this?